masterbase/app.py

```diff
@@ -101,17 +102,16 @@ def close_session(request: Request, api_key: str) -> dict[str, bool]:
 
 
 @post("/late_bytes", guards=[valid_key_guard, user_not_in_session_guard], sync_to_thread=False)
-def late_bytes(request: Request, api_key: str, data: dict[str, str]) -> dict[str, bool]:
+def late_bytes(request: Request, api_key: str, data: LateBytesBody) -> dict[str, bool]:
     """Add late bytes to a closed demo session..
 
     Returns:
         {"late_bytes": True}
     """
     engine = request.app.state.engine
     current_time = datetime.now().astimezone(timezone.utc)
-    late_bytes = bytes.fromhex(data["late_bytes"])
     steam_id = steam_id_from_api_key(engine, api_key)
-    late_bytes_helper(engine, steam_id, late_bytes, current_time)
+    late_bytes_helper(engine, steam_id, data.converted_late_bytes, current_time)
```

Comment:
If this can throw anything, might want to make sure we catch/re-raise gracefully instead of `500`. If we're already doing that in `late_bytes_helper` then please disregard

Reply:
😎
```py
response = requests.post(
    f"http://{app_host}/late_bytes?api_key={api_key}", json={"late_bytes": "foo"}
)
```
>>>

```
{'status_code': 400, 'detail': 'Validation failed for POST /late_bytes?api_key=some_api_key', 'extra': [{'message': 'Value error, non-hexadecimal number found in fromhex() arg at position 1'}]}
```

Pydantic + litestar rocks